### PR TITLE
Upgrade testbed image to Ubuntu 24.04

### PR DESCRIPTION
**Pull request by lindenb1**
cf. https://github.com/osism/issues/issues/1028

### CODE REVIEW

**Comment by gndrmnn** [comment]
Needed: https://github.com/osism/terraform-base/pull/54

**Comment by gndrmnn** [comment]
https://github.com/osism/terraform-base/blob/c6062af528396af20054167036d497678fa07202/testbed-default/manager.tf#L71-L121

**Comment by sbstnnmnn** [comment]
Pull of kolla images fails.
e.g. `osism apply -a pull barbican`:

```
TASK [service-images-pull : barbican | Pull images] ****************************
Wednesday 22 May 2024  10:28:38 +0000 (0:00:01.988)       0:00:10.195 ********* 
[...]
failed: [testbed-node-2.testbed.osism.xyz] (item=barbican-api) => {"ansible_loop_var": "item", "attempts": 3, "changed": true, "item": {"key": "barbican-api", "value": {"container_name": "barbican_api", "dimensions": {}, "enabled": true, "group": "barbican-api", "haproxy": {"barbican_api": {"enabled": "yes", "external": false, "listen_port": "9311", "mode": "http", "port": "9311", "tls_backend": "no"}, "barbican_api_external": {"enabled": "yes", "external": true, "external_fqdn": "api.testbed.osism.xyz", "listen_port": "9311", "mode": "http", "port": "9311", "tls_backend": "no"}}, "healthcheck": {"interval": "30", "retries": "3", "start_period": "5", "test": ["CMD-SHELL", "healthcheck_curl http://192.168.16.12:9311"], "timeout": "30"}, "image": "nexus.testbed.osism.xyz:8192/osism/barbican-api:2023.2", "volumes": ["/etc/kolla/barbican-api/:/var/lib/kolla/config_files/:ro", "/etc/localtime:/etc/localtime:ro", "/etc/timezone:/etc/timezone:ro", "barbican:/var/lib/barbican/", "kolla_logs:/var/log/kolla/", ""]}}, "msg": "'Traceback (most recent call last):
  File \"/usr/lib/python3/dist-packages/docker/api/client.py\", line 214, in _retrieve_server_version
    return self.version(api_version=False)[\"ApiVersion\"]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File \"/usr/lib/python3/dist-packages/docker/api/daemon.py\", line 181, in version
    return self._result(self._get(url), json=True)
                        ^^^^^^^^^^^^^^
  File \"/usr/lib/python3/dist-packages/docker/utils/decorators.py\", line 46, in inner
    return f(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File \"/usr/lib/python3/dist-packages/docker/api/client.py\", line 237, in _get
    return self.get(url, **self._set_request_timeout(kwargs))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File \"/usr/lib/python3/dist-packages/requests/sessions.py\", line 602, in get
    return self.request(\"GET\", url, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File \"/usr/lib/python3/dist-packages/requests/sessions.py\", line 589, in request
    resp = self.send(prep, **send_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File \"/usr/lib/python3/dist-packages/requests/sessions.py\", line 703, in send
    r = adapter.send(request, **kwargs)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File \"/usr/lib/python3/dist-packages/requests/adapters.py\", line 486, in send
    resp = conn.urlopen(
           ^^^^^^^^^^^^^
  File \"/usr/lib/python3/dist-packages/urllib3/connectionpool.py\", line 791, in urlopen
    response = self._make_request(
               ^^^^^^^^^^^^^^^^^^^
  File \"/usr/lib/python3/dist-packages/urllib3/connectionpool.py\", line 497, in _make_request
    conn.request(
TypeError: HTTPConnection.request() got an unexpected keyword argument \\'chunked\\'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File \"/tmp/ansible_kolla_container_payload_ynjl248_/ansible_kolla_container_payload.zip/ansible/modules/kolla_container.py\", line 417, in main
    cw = DockerWorker(module)
         ^^^^^^^^^^^^^^^^^^^^
  File \"/tmp/ansible_kolla_container_payload_ynjl248_/ansible_kolla_container_payload.zip/ansible/module_utils/kolla_docker_worker.py\", line 39, in __init__
    self.dc = get_docker_client()(**options)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File \"/usr/lib/python3/dist-packages/docker/api/client.py\", line 197, in __init__
    self._version = self._retrieve_server_version()
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File \"/usr/lib/python3/dist-packages/docker/api/client.py\", line 221, in _retrieve_server_version
    raise DockerException(
docker.errors.DockerException: Error while fetching server API version: HTTPConnection.request() got an unexpected keyword argument \\'chunked\\'
'"}
```

File: ansible/manager-part-0.yml [hunk 1]
```diff
@@ -19,10 +19,10 @@
     ansible_galaxy: "{{ venv_path }}/bin/ansible-galaxy"
 
   tasks:
-    - name: Fail if Ubuntu version is lower than 22.04
+    - name: Fail if Ubuntu version is lower than 24.04
```
[the remark this answers is not in the document]
Comment (lindenb1) [reply ansible/manager-part-0.yml]: Rolled back

**Pull request closed without merging**